src/main/scala/cromwell/engine/backend/jes/JesAttributes.scala

```diff
@@ -0,0 +1,41 @@
+package cromwell.engine.backend.jes
+
+import java.net.URL
+
+import com.typesafe.config.{Config, ConfigFactory}
+import cromwell.util.ConfigUtil._
+
+case class JesAttributes(applicationName: String,
+                         project: String,
+                         executionBucket: String,
+                         endpointUrl: URL,
+                         authMode: GcsAuthMode,
+                         docker: Option[DockerAuthInformation])
+object JesAttributes {
+
+  def apply(): JesAttributes = {
+    val jesConf = ConfigFactory.load.getConfig("backend").getConfig("jes")
+    val refConf: Config = ConfigFactory.parseResources("jes.conf")
+
+    jesConf.checkValidWrapped(refConf, "Jes")
+    jesConf.warnNotRecognized(refConf, "Jes", "dockerAccount", "dockerToken")
+
+    val applicationName = jesConf.getString("applicationName")
+    val project = jesConf.getString("project")
+    val executionBucket = jesConf.getString("baseExecutionBucket")
+    val endpointUrl = jesConf.getURL("endpointUrl")
+    val authMode = GcsAuthMode.fromString(jesConf.getString("authenticationMode"))
+    val docker = for {
+      account <- jesConf.getStringOption("dockerAccount")
+      token <- jesConf.getStringOption("dockerToken")
+    } yield DockerAuthInformation(account, token)
+
```

Comment:
I see your point, I'd say in some sense checkValid just removes the need to have several
```
def validateString(key: String): ValidationNel[String, String] = {
   try {
    config.getString(key).successNel
   } catch {
    case e: ConfigException.Missing => s"Missing key $key".failureNel
    case e: ConfigException.WrongType => s"Wrong type for key $key".failureNel
   }
}

def validateBoolean(key: String): ValidationNel[String, Boolean] = {
   try {
    config.getBoolean(key).successNel
   } catch {
    case e: ConfigException.Missing => s"Missing key $key".failureNel
    case e: ConfigException.WrongType => s"Wrong type for key $key".failureNel
   }
}
...
```
and use them on every configuration key.
We can do everything with Validations, it just seemed like some of this work is already implemented by `checkValid` which is why I thought it was worth using it.
Honestly I don't really have a strong opinion on this, Validations seems to be a bit heavy and "overkill" kind of tool to me only to check a few keys exist in a conf file, but maybe it's not.